yellowbrick/classifier/prcurve.py

```diff
@@ -133,8 +133,9 @@ class PrecisionRecallCurve(ClassificationScoreVisualizer):
         In the multiclass case, a mapping of class/metric to recall array.
 
 
-    Example
-    -------
+    Examples
+    --------
```

Comment:
Fixes a warning "unknown section Example" (all other changes are the linter removing whitespace)